GameServerLib/GameObjects/AttackableUnits/AI/BaseTurret.cs

```diff
@@ -200,7 +202,6 @@ public override void Update(float diff)
             }
 
             base.Update(diff);
-            Replication.Update();
```

Comment:
Why replication is not updated here? Is it now in AI base class?

Reply:
It should work fine as the Replication is overridden with the proper type, and each Replication update occurs in Update().